experiments/openebs/openebs-pool-network-loss/openebs_pool_network_loss_ansible_logic.yml

```diff
@@ -0,0 +1,174 @@
+- hosts: localhost
+  connection: local
+
+  vars:
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_pvc: "{{ lookup('env','APP_PVC') }}"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    c_experiment: "openebs-pool-network-loss"
+    c_interval: "5"
+    data_persistence: "{{ lookup('env','DATA_PERSISTENCE') }}"
+    lib_image: "{{ lookup('env','LIB_IMAGE') }}"
+    liveness_label: "{{ lookup('env','LIVENESS_APP_LABEL') }}"
+    liveness_namespace: "{{ lookup('env','LIVENESS_APP_NAMESPACE') }}"
+    packet_loss_perc: "{{ lookup('env','NETWORK_PACKET_LOSS_PERCENTAGE') }}"  
+    openebs_ns: "{{ lookup('env','OPENEBS_NAMESPACE') }}"
+
+  vars_files:
+    - /mnt/parameters.yml
+    - /experiments/openebs/openebs_components.yml
+  
+  tasks:
+    - block:
+      - include_tasks: /utils/runtime/create_testname.yml  
+
+      - name: Identify the chaos util to be invoked 
+        template:
+          src: chaosutil.j2
+          dest: chaosutil.yml
+
+      - name: Identify the data consistency util to be invoked
+        template:
+          src: data_persistence.j2
+          dest: data_persistence.yml  
+
+      - name: Including chaos util for running the experiment
+        include_vars:
+          file: chaosutil.yml
+      
+      - name: Including data persistence util to check data persistency
+        include_vars:
+          file: data_persistence.yml
+      
+      # GENERATE EXPERIMENT RESULT NAME AND CREATE CHAOS RESULT RESOURCE
+      - block:
+          - name: Construct chaos result name (experiment_name)
+            set_fact:
+              c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
+            when: lookup('env','CHAOSENGINE')
+
+          - include_tasks: /utils/runtime/update_chaos_result_resource.yml
+            vars:
+              status: 'SOT'
+              namespace: "{{ a_ns }}"
+
+      # DISPLAY APP INFORMATION
+      - name: Display the app information passed via the test job
+        debug:
+          msg:
+            - "The application info is as follows:"
+            - "Namespace    : {{ a_ns }}"
+            - "Label        : {{ a_label }}"
+            - "PVC          : {{ a_pvc }}"
+            # - "StorageClass : {{ sc }}"
+            
+      # Pre req
+      - block:
```

Comment:
Can you please add this step in openebs_pool_network_loss_ansible_prerequisites.yml file?